src/main/kotlin/no/nav/familie/ef/mottak/repository/domain/EttersendingDb.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.familie.ef.mottak.repository.domain
+
+import no.nav.familie.ef.mottak.encryption.FileCryptoConverter
+import no.nav.familie.ef.mottak.encryption.StringValCryptoConverter
+import java.time.LocalDateTime
+import java.util.*
+import javax.persistence.*
```

Comment:
Vi har en felles code-style vi bruker for alle kotlin apper i teamfamilie: Clone repoet https://github.com/navikt/familie-settings og legg til `teamfamilie-code-style.xml` under codestyle i intellij.

Etter du har gjort dette kan du kjøre OPTION + CMD + L for å reformatere koden, og CTRL + OPTION + O for å fikse imports. 

Importsene over skal da gjerne komme med komplett path :)

Reply:
😎